src/constants/tokens/vBep/mainnet.ts

```diff
@@ -172,5 +172,25 @@ export const MAINNET_VBEP_TOKENS = {
     symbol: 'vWBETH',
     underlyingToken: MAINNET_TOKENS.wbeth,
   } as VToken,
-  // TODO: add tokens from isolated pools
+
+  // Isolated assets
+  // Pool 1 – Stable Coins
+  '0xca2d81aa7c09a1a025de797600a7081146dceed9': {
+    address: '0xca2d81aa7c09a1a025de797600a7081146dceed9',
+    decimals: 8,
+    symbol: 'vHAY',
```

Comment:
Where do we use this symbol? Because the real symbol of the vToken is `vHAY_Stablecoins`. The rule for the symbols in IL is: `v` + `<underlying token symbol>` + `_<pool name>`. Do we need to consider it?

Reply:
It's only used for display on the frontend, we never actually use the symbol in any transaction (we always use the addres)